### ISSUE TITLE
Packethost gem requires Ruby 2.1+

### ISSUE THREAD

**Issue by ianwalter** [kontena/kontena-plugin-packet#6]
```bash
kontena packet master create \
--token $PACKET_API_TOKEN \
--project $PACKET_PROJECT_ID \
--type baremetal_0 \
--facility ewr1
Generating self-signed SSL certificate... done
/Library/Ruby/Gems/2.0.0/gems/packethost-0.0.7/lib/packet/entity/serialization.rb:20:in `to_hash': undefined method `to_h' for #<Array:0x007fdc7b1d6528> (NoMethodError)
        from /Library/Ruby/Gems/2.0.0/gems/kontena-plugin-packet-0.1.1/lib/kontena/machine/packet/master_provisioner.rb:46:in `run!'
        from /Library/Ruby/Gems/2.0.0/gems/kontena-plugin-packet-0.1.1/lib/kontena/plugin/packet/master/create_command.rb:24:in `execute'
        from /Library/Ruby/Gems/2.0.0/gems/clamp-1.0.0/lib/clamp/command.rb:68:in `run'
        from /Library/Ruby/Gems/2.0.0/gems/clamp-1.0.0/lib/clamp/subcommand/execution.rb:11:in `execute'
        from /Library/Ruby/Gems/2.0.0/gems/clamp-1.0.0/lib/clamp/command.rb:68:in `run'
        from /Library/Ruby/Gems/2.0.0/gems/clamp-1.0.0/lib/clamp/subcommand/execution.rb:11:in `execute'
        from /Library/Ruby/Gems/2.0.0/gems/clamp-1.0.0/lib/clamp/command.rb:68:in `run'
        from /Library/Ruby/Gems/2.0.0/gems/clamp-1.0.0/lib/clamp/subcommand/execution.rb:11:in `execute'
        from /Library/Ruby/Gems/2.0.0/gems/clamp-1.0.0/lib/clamp/command.rb:68:in `run'
        from /Library/Ruby/Gems/2.0.0/gems/clamp-1.0.0/lib/clamp/command.rb:133:in `run'
        from /Library/Ruby/Gems/2.0.0/gems/kontena-cli-0.15.1/bin/kontena:16:in `<top (required)>'
        from /usr/local/bin/kontena:23:in `load'
        from /usr/local/bin/kontena:23:in `<main>'
```

**Comment by kke** [comment]
The to_h method is only available in Ruby version 2.1 or higher.

Workaround: Update your ruby if possible.

**Comment by ianwalter** [comment]
Ok sure. I think you guys should update the requirements in the Quick Start guide though. Thanks.

**Comment by kke** [comment]
I think we will try to modify it to run on 2.0

**Comment by kke** [comment]
Waiting for [packet-rb PR](https://github.com/packethost/packet-rb/pull/11)

**Comment by kke** [comment]
Fixed in version 0.1.2. 

```
$ gem install kontena-plugin-packet 
```
